### ISSUE TITLE
About gradient accumulation

### ISSUE THREAD

**Issue by XinYu-Andy** [threestudio-project/threestudio#59]
Hi! Thanks for your great effort!
I am currently trying your repo. Following your tips, I am trying to use gradient accumulation to get a bigger batch size. However, when I use this config, there is an error occurred:
```
MisconfigurationException: Automatic gradient accumulation is not supported for manual optimization. Remove
`Trainer(accumulate_grad_batches=8)` or switch to automatic optimization.
```
Have you meet this before and have some advice to solve this?

**Comment by bennyguo** [comment]
Hi! Are you experimenting with the Fantasia3D model? I just pushed a fix enabling automatic optimization for Fantasia3D, now it should be working with gradient accumulation.

**Comment by XinYu-Andy** [comment]
> Hi! Are you experimenting with the Fantasia3D model? I just pushed a fix enabling automatic optimization for Fantasia3D, now it should be working with gradient accumulation.

Exactly! Thank you very much, I will pull it and try it again
